src/Models/LagrangianParticleTracking/lagrangian_particle_advection.jl

```diff
@@ -118,10 +118,14 @@ given `velocities`, time-step `Δt, and coefficient of `restitution`.
 
     current_particle_indices = (i, j, k)
 
+    up = interpolate(X, velocities.u, (f, c, c), grid)
+    vp = interpolate(X, velocities.v, (c, f, c), grid)
+    wp = interpolate(X, velocities.w, (c, c, f), grid)
```

Comment:
I think a clearer notation would be to define this as `u, v, w` or `ui, vi, wi` and the particle velocities as `up, vp, wp`

Reply:
If I understand correctly, the interpolation yields the fluid velocities, but the particles can add or replace these with their own values. The replaced values are the "particle velocities". non-inertial particles have `up=uf`. This notation is confusing because the flow velocity is called `up` which suggests "particle"... but the explicit purpose of this PR is to allow non-flow particle velocities

Reply:
maybe we can call the fluid velocities `uf`